en/assembly/quick_start_pixhawk_v6x.md

```diff
@@ -0,0 +1,138 @@
+# Pixhawk V6X Wiring Quick Start
+
+This quick start guide shows how to power the Pixhawk V6X<sup>&reg;</sup> flight controller and connect its most important peripherals.
+
+## Wiring Chart Overview {#wiring-Chart-Overview}
+
+The image below shows how to connect the most important sensors and peripherals (except the motor and servo outputs). We'll go through each of these in detail in the following sections.
+
+![wiring](../../assets/flight_controller/pixhawk_v6x/quickstart_01_en.jpg)
+
+| Main interface | **Function** |
+| :--- | :--- |
+| POWER C1 | Please connect CAN PMU SE to this interface; this interface is connected to UAVCAN power module |
+| POWER C2 | Please connect CAN PMU SE to this interface; this interface is connected to UAVCAN power module |
+| POWER 1 | Connect SMbus(I2C) power module |
+| POWER 2 | Connect SMbus(I2C) power module |
+| GPS&SAFETY |Connect Neo series GPS/C-RTK 9PS, including GPS, safety switch, buzzer interface.|
+| GPS2 |Connect GPS/RTK module|
+| UART 4 | Support user customization |
+| TELEM1/TELME2/TELEM3 | Connect telemetry or mavlink devices|
+| TF CARD | SD card for log storage (card pre-inserted in factory). |
+| M1~M8 | PWM output from IO , connecting to ESC and Servo|
+| A1~A8 | Form FMU,Can be defined as PWM/GPIO; supports Bdshot; used to connect camera shutter/hot shoe, servo, etc. |
+| USB | Connect to a computer for communication between the flight controller and the computer, such as loading firmware. |
+| CAN1/CAN2 | Connect Dronecan/UAVCAN devices such as NEO3 Pro. |
+| DSM/SUB/RSSI | Includes DSM, SBUS, RSSI signal input interface, DSM interface can be connected to DSM satellite receiver, SBUS interface to SBUS remote control receiver, RSSI for signal strength return module |
+| PPM | Connecting the PPM RC Receiver |
+| ETH |Ethernet interface, you can connect Ethernet devices such as task computers.|
+|AD&IO| There are two analog inputs (ADC3.3/ADC6.6); generally not used|
+
+## Vehicle Front
+
+> **NOTE** If the controller cannot be mounted in the recommended/default orientation (e.g. due to space constraints) you will need to configure the autopilot software with the orientation that you actually used: [Flight Controller Orientation](https://docs.px4.io/v1.9.0/en/assembly).
```

Comment:
If linking internally, please use an internal link. Linking externally will eventually fail when we delete the old docs.

```suggestion
:::note
If the controller cannot be mounted in the recommended/default orientation (e.g. due to space constraints) you will need to configure the autopilot software with the orientation that you actually used: [Flight Controller Orientation](../config/flight_controller_orientation.md).
:::
```